index.js

```diff
@@ -871,9 +880,19 @@ RingPop.prototype.handleOrProxyAll =
                     dest: dest
                 }));
             }
-            responses[dest] = res;
-            return responses;
-        }, {});
+        });
+
+        function onResponse(err, data, dest) {
+            if (data) {
```

Comment:
you cant just add a dest & keys field to data. You want `{ body: data, dest, keys }`